infra/README.md

```diff
@@ -1,9 +1,14 @@
 # Getting Started With Terraform
 
 ## Install Terraform CLI
-&nbsp;&nbsp;Terraform is an infrastructure as code (IaC) tool that allows you to build, change, and version infrastructure safely and efficiently. This includes both low-level components like compute instances, storage, and networking, as well as high-level components like DNS entries and SaaS features. Install the terraform commmand line tool by follow the instructions found here:
+&nbsp;&nbsp;Terraform is an infrastructure as code (IaC) tool that allows you to build, change, and version infrastructure safely and efficiently. This includes both low-level components like compute instances, storage, and networking, as well as high-level components like DNS entries and SaaS features.
 
-- [Install Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+You may need different versions of Terraform since different projects may require different versions of Terraform. The best way to manage Terraform versions is with [Terraform Version Manager](https://github.com/tfutils/tfenv).
+
+```bash
+brew install tfenv
```

Comment:
Do need to add a blurb about what brew is?

Reply:
added a link to homebrew, but otherwise i think homebrew is common enough i don't think we need to explain it.